config/install/field.field.paragraph.lesson_section.field_lesson_section_content.yml

```diff
@@ -78,6 +80,9 @@ settings:
       lesson_embedded_video:
         enabled: true
         weight: -40
+      lesson_footnotes:
```

Comment:
Don't put content paragraphs below question paragraphs - it creates some mess in the UI when add new paragraphs. The rest of the PR looks good to me.

Reply:
Done.